core/src/main/resources/hudson/model/AbstractBuild/tasks.jelly

```diff
@@ -27,7 +27,6 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:p="/lib/hudson/project">
-      <l:task contextMenu="false" href="${it.upUrl}" icon="icon-up icon-md" title="${%Back to Project}"/>
```

Comment:
Could then deprecate https://github.com/jenkinsci/jenkins/blob/082634cf1e4ef91cac8c82815e52046083722e36/core/src/main/java/hudson/model/AbstractBuild.java#L263-L277 I think. (Still used by some plugins, so cannot be deleted yet.)

Reply:
Done, cheers! 👍